### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by trishorts**

### CODE REVIEW

**Review by nbollis — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing to Transcriptase
```
Comment (acesnik) [inline CONTRIBUTING.md:1]: Change to mzLib?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/briandk/transcriptase-atom/issues)
```
Comment (acesnik) [inline CONTRIBUTING.md:26]: Correct link here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (acesnik) [inline CONTRIBUTING.md:27]: Missing link

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Write bug reports with detail, background, and sample code
+[This is an example](http://stackoverflow.com/q/12488905/180626) of a bug report I wrote, and I think it's not a bad model. Here's [another example from Craig Hockenberry](http://www.openradar.me/11905408), an app developer whom I greatly respect.
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [My stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that *anyone* with a base R setup can run to reproduce what I was seeing
```
Comment (acesnik) [inline CONTRIBUTING.md:37]: Change to C#?

**Review by acesnik — CHANGES_REQUESTED** [review]

**Review by nbollis — APPROVED** [review]

**Review by Alexander-Sol — APPROVED** [review]

**Review by acesnik — APPROVED** [review]

**Pull request merged**